ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageManager.java

```diff
@@ -93,9 +96,27 @@ public void setBorderWidth(ReactImageView view, float borderWidth) {
     view.setBorderWidth(borderWidth);
   }
 
-  @ReactProp(name = "borderRadius")
-  public void setBorderRadius(ReactImageView view, float borderRadius) {
-    view.setBorderRadius(borderRadius);
+  @ReactPropGroup(names = {
+      ViewProps.BORDER_RADIUS,
+      ViewProps.BORDER_TOP_LEFT_RADIUS,
+      ViewProps.BORDER_TOP_RIGHT_RADIUS,
+      ViewProps.BORDER_BOTTOM_RIGHT_RADIUS,
+      ViewProps.BORDER_BOTTOM_LEFT_RADIUS
+  }, defaultFloat = CSSConstants.UNDEFINED)
+  public void setBorderRadius(ReactImageView view, int index, float radius) {
+    if (!CSSConstants.isUndefined(radius)) {
+      radius = PixelUtil.toPixelFromDIP(radius);
+    }
+
+    if (index == 0) {
+      view.setBorderRadius(0, radius);
+      view.setBorderRadius(1, radius);
+      view.setBorderRadius(2, radius);
+      view.setBorderRadius(3, radius);
+      return;
```

Comment:
IMO `if {} else {}` construct would be much cleaner to read in this case (also less error prone for people making changes in the future, as it's easy to overlook that return statement)